app/build.gradle

```diff
@@ -121,6 +121,21 @@ android {
             }
         }
 
+        mainnet_staging {
```

Comment:
Should we rename this to `mainnet_internal` or alternatively, rename `internal` to `staging` ?

Reply:
mainnet_staging is more descriptive. internal version could be renamed to rinkeby_staging, but I would not do it now. Currently, everyone is aware of what internal version is composed of.